templates/layout.html

```diff
@@ -58,7 +58,7 @@
   <div class="container">
     {% block body %}{% endblock %}
   </div>
-  <footer class="text-center text-lg-start bg-light text-muted">
+  <footer class="text-center text-lg-start bg-light text-muted footer">
```

Comment:
Footer class for the footer element seems redundant. Can't the CSS be attached to the element already?

Reply:
Yes, it can. It's just hard to know whether people prefer direct element selectors or classes always. I have changed it to use the `footer`, as a direct selector.